### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by ShahierNashaat**
Pull request for review of the style of the leaderboard project using CSS.

Style the webpage.

Fix linter error.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -20,9 +20,16 @@ <h1>Leaderboard</h1>
...
+            <div class="name-area">
+                <input id="name" type="text" placeholder="your name">
+                <span class="name-warning display-none">* please enter the name</span>
+            </div>
+            <div class="score-area">
+                <input id="score" type="number" placeholder="your score">
+                <span class="score-warning display-none">* please enter the score</span>
+                <span class="minus-score-warning display-none">* the score must be more than or equal zero</span>
+            </div>
+            <button type="submit" class="submit-btn">Submit</button>
```
Comment (Mihndim2020) [inline src/index.html:32]: - [OPTIONAL] Please each input is displayed twice, kindly fix that
SHA
.

File: src/style.css [hunk 2]
```diff
@@ -51,5 +67,35 @@ li:nth-child(even) {
 }
 
 button {
```
Comment (Mihndim2020) [inline src/style.css:69]: - [ ] The refresh button is not properly aligned, please kindly fix that. 

***The wireframe***
sha2

***Your implementation***
sha3

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: The app is running
:heavy_check_mark: Good job passing the linter checks

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments below and  under the review._

- [ ] Please kindly update your readme file to reflects the changes implemented for this milestone. Please checkout the screenshot below and implement the changes. 
sha04

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Shahier, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**